pkg/generate/generate.go

```diff
@@ -306,19 +315,6 @@ func manageData(log logr.Logger, kind, namespace, name string, data map[string]i
 }
 
 func manageClone(log logr.Logger, kind, namespace, name string, clone map[string]interface{}, client *dclient.Client, resource unstructured.Unstructured) (map[string]interface{}, ResourceMode, error) {
-	// check if resource to be generated exists
```

Comment:
Please have a look on this function carefully. It can break the functionality

Reply:
@realshuting did you found anything here that can break before sync functionality ?